networkx/algorithms/connectivity/connectivity.py

```diff
@@ -325,8 +325,9 @@ def neighbors(v):
     kwargs = dict(flow_func=flow_func, auxiliary=H, residual=R)
 
     # Pick a node with minimum degree
-    degree = G.degree()
+    degree = dict(G.degree())
     minimum_degree = min(degree.values())
+    # TODO
```

Comment:
Here's another way -- not sure what is best.

    minimum_degree = min( d for n,d in G.degree() )